index.html

```diff
@@ -27,12 +27,31 @@
 			<nav class="navbar">
 				<a href="#" class="navbar__logo">Lusindiso Ntanjana</a>
 				<ul class="navbar__menu--small">
-					<li>
+					<li id="modal-btn">
 						<a href="#" class="menubar">
 							<i class="fa-solid fa-bars"></i>
 						</a>
 					</li>
 				</ul>
+				<div id="modal" class="modal">
+					<div class="close">
+						<img src="./img/close.svg" alt="close button" id="closeModal" class="close-modal">
+					</div>
+					<ul class="modal-menu">
+						<li class="modal-menu__item">
+							<a href="#portfolio" class="modal-menu__item--link modalLink">Portoflio</a>
+						</li>
+						<li class="modal-menu__item">
+							<a href="#about" class="modal-menu__item--link modalLink">About</a>
+						</li>
+						<li class="modal-menu__item">
+							<a href="#contact" class="modal-menu__item--link modalLink">Contact</a>
+						</li>
+					</ul>
+					<div class="modal__bar">
+						<img src="./img/Shape.svg" alt="" />
+					</div>
+				</div>
```

Comment:
- [ ] The Menu is still displayed on the desktop version. Kindly hide the Menu on the desktop version so your project is complete.

<img width="940" alt="image" src="https://user-images.githubusercontent.com/77190203/161562057-4fdc2ae7-2c70-466a-93a1-fadb381b2183.png">